inspire/base/static/less/format/detailed-record.less

```diff
@@ -0,0 +1,279 @@
+/**
+ *
+ * This file is part of INSPIRE.
+ * Copyright (C) 2015 CERN.
+
+ * INSPIRE is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * Invenio is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Invenio; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+ @import (reference) "../inspire.less";
+
+ @teal: #EDF3F3;
+ @grey: #7E8D8E;
+ @purple: #293E51;
+
+ #topnav {
+ 	min-height: 65px;
```

Comment:
Is this correct or do I need to edit it? Is there a difference between 4 spaces and 1 tab?

![Indentation]
(http://i.imgur.com/HSjdJup.png)